app/src/main/res/drawable/wind.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.wind` appears to be unused
 Unused resources make applications larger and slow down builds.